addons/pos_mollie/static/src/app/payment_mollie.js

```diff
@@ -0,0 +1,176 @@
+/** @odoo-module */
+
+import { _t } from "@web/core/l10n/translation";
+import { PaymentInterface } from "@point_of_sale/app/payment/payment_interface";
+import { ErrorPopup } from "@point_of_sale/app/errors/popups/error_popup";
+import { ConfirmPopup } from "@point_of_sale/app/utils/confirm_popup/confirm_popup";
+
+export class PaymentMollie extends PaymentInterface {
+    /**
+     * @override
+     */
+    setup() {
+        super.setup(...arguments);
+        this.paymentLineResolvers = {};
+    }
+
+    /**
+     * @override
+     */
+    send_payment_request(cid) {
+        super.send_payment_request(cid);
+        return this._make_mollie_payment(cid);
+    }
+
+    /**
+     * @override
+     *
+     * At the moment, Mollie POS payments are no cancellable from the API call.
+     * It can be only cancelled from the terminal itself. If you cancel the
+     * transaction from the terminal, we get webhook call for status update and
+     * `handleMollieStatusResponse` will handle cancellation.
+     *
+     * But in case you want to cancel from odoo pos (e.g terminal switched off during transaction)
+     * We show the dialog to guide user in correct direction.
+     */
+    async send_payment_cancel(order, cid) {
+        const { confirmed } = await this.env.services.popup.add(ConfirmPopup, {
+            title: _t('Cancel mollie payment'),
+            body: _t('First cancel transaction on POS device. Only use force cancel if that fails'),
```

Comment:
this doesn't feel extremely intuitive... The user will be tempted to click on the confirmation button, which is not what you want in this case

Reply:
Yes you are right.

I have removed message instead, i have cancelled payment and show the message `Mollie payments also need manual cancellation on the payment terminal.` 

Is it ok ?

Reply:
Nice!